src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -26,27 +27,13 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
 
   def withActiveUser(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
-      val requireActiveUserIO = for {
-        user <- getSamUser(directoryDAO, registrationDAO, oidcHeaders, samRequestContext)
-      } yield {
-        // service account users do not need to accept ToS
-        val tosStatusAcceptable = tosService.isTermsOfServiceStatusAcceptable(user) || SAdomain.matches(user.email.value)
-        if (!tosStatusAcceptable) {
-          throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.Unauthorized, "User has not accepted the terms of service."))
-        }
-        if (!user.enabled) {
-          throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.Unauthorized, "User is disabled."))
-        }
-
-        user
-      }
-      requireActiveUserIO.unsafeToFuture()
```

Comment:
this moved to the `StandardSamUserDirectives` object below so it could be called from tests